modules/mongodb/mongodb.go

```diff
@@ -44,6 +55,15 @@ func (m *Mongo) Init() bool {
 		return false
 	}
 
+	if !m.Databases.Empty() {
+		mMatcher, err := m.Databases.Parse()
+		if err != nil {
+			m.Errorf("error on creating 'databases' matcher : %v", err)
+			return false
+		}
+		m.databasesMatcher = matcher.WithCache(mMatcher)
```

Comment:
```suggestion
		m.databasesMatcher = mMatcher
```

I don't think we need cache here, it should be pretty fast w/o it.

Reply:
I have removed the cache here